### PR TITLE
Fix argument checking for b (break) command in python debugger

### PR DESCRIPTION
**Pull request by matedabis**
Signed-off-by: Mate Dabis <mdabis@inf.u-szeged.hu>

### CODE REVIEW

**Comment by matedabis** [comment]
@clover2123 
I created a new testcase for this patch

File: tools/debugger/tests/do_break_no_arg.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ *  Lesser General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Lesser General Public
+ *  License along with this library; if not, write to the Free Software
+ *  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301
+ *  USA
+ */
+
+var o = null
```
Comment (zherczeg) [inline tools/debugger/tests/do_break_no_arg.js:20]: I don't think we need this complex testcase for this. Some simple js should be enough,

**Review by clover2123 — APPROVED** [review]
BTW,  sign-off in your commit message includes `JerryScript-DCO-1.0-Signed-off-by` relavant to jerryscript project.

**Comment by matedabis** [comment]
modified sign-off

**Review by ksh8281 — APPROVED** [review]

**Pull request merged**